phpBB/phpbb/event/php_exporter.php

```diff
@@ -264,7 +264,23 @@ public function crawl_php_file($file)
 
 					// Find event description line
 					$description_line_num = $this->find_description();
-					$description = substr(trim($this->file_lines[$description_line_num]), strlen('* '));
+					$description_lines = array();
+
+					while (true)
+					{
+						$description_line = substr(trim($this->file_lines[$description_line_num]), strlen('* '));
+
+						// Reached end of description if line is empty or a tag
+						if (!strlen($description_line) || $description_line[0] == '@')
```

Comment:
imo we could take everything between the first line and the first tag (or the end of the comment to be safe in case of mal formed comment)

Reply:
Extracting everything until the first tag is a good idea.

Extracting the entire comment block and then removing all tags should be done in a separate PR, since all the other code (finding tags etc) expects each line in a comment to start with a `*` as well.

Reply:
On second thought, do we actually want to extract everything before the first tag? For example, [this event description](https://github.com/phpbb/phpbb/blob/release-3.2.2/phpBB/common.php#L152-L159) is quite long and would clutter the [events list](https://wiki.phpbb.com/Event_List).

Reply:
tbh I think it is better to extract everything because there is little chances that the comment will be read otherwise

Reply:
will you have time to work on that?

Reply:
Yes. :smile: I tested extracting the description from a block without tags and from a block without a description and in both cases it threw an exception before the code affected by this PR is run, so I think we can assume that all the information is there.